### ISSUE TITLE
error on testing the package 

### ISSUE THREAD

**Issue by aheravi** [dami82/mutSignatures#17]
Hi,
I'm getting an error in testing your package based on the tutorial you've provided here:
https://cran.r-project.org/web/packages/mutSignatures/vignettes/get_sarted_with_mutSignatures.html

```
# Attach context
x <- attachContext(mutData = x,chr_colName = "CHROM",start_colName = "POS", end_colName = "POS",nucl_contextN = 3,BSGenomeDb = hg19)

An error has occurred!
```
And here is part of my sessioninfo():
```
other attached packages:
 [1] mutSignatures_2.1.1               foreach_1.5.2                    
 [3] BSgenome.Hsapiens.UCSC.hg19_1.4.3 BSgenome_1.62.0                  
 [5] rtracklayer_1.54.0                Biostrings_2.62.0                
 [7] XVector_0.34.0                    GenomicRanges_1.46.1             
 [9] GenomeInfoDb_1.30.1               IRanges_2.28.0                   
[11] S4Vectors_0.32.3                  BiocGenerics_0.40.0              
[13] gridExtra_2.3                     ggplot2_3.4.0                    
[15] kableExtra_1.3.4                  reshape2_1.4.4                   
[17] dplyr_1.0.8       
```

Could you please comment on that?
Thanks,
Ali

**Comment by dami82** [comment]
You are using an outdated version of mutSignatures (2.1.1). Please install the latest version from github (2.1.5) and try again. Let me know if the error persists.Best regards,Damiano——Damiano FantiniSent from my iPhoneOn Feb 2, 2023, at 18:39, aheravi ***@***.***> wrote:﻿
Hi,
I'm getting an error in testing your package based on the tutorial you've provided here:
https://cran.r-project.org/web/packages/mutSignatures/vignettes/get_sarted_with_mutSignatures.html
# Attach context
x <- attachContext(mutData = x,chr_colName = "CHROM",start_colName = "POS", end_colName = "POS",nucl_contextN = 3,BSGenomeDb = hg19)

An error has occurred!

And here is part of my sessioninfo():
other attached packages:
 [1] mutSignatures_2.1.1               foreach_1.5.2                    
 [3] BSgenome.Hsapiens.UCSC.hg19_1.4.3 BSgenome_1.62.0                  
 [5] rtracklayer_1.54.0                Biostrings_2.62.0                
 [7] XVector_0.34.0                    GenomicRanges_1.46.1             
 [9] GenomeInfoDb_1.30.1               IRanges_2.28.0                   
[11] S4Vectors_0.32.3                  BiocGenerics_0.40.0              
[13] gridExtra_2.3                     ggplot2_3.4.0                    
[15] kableExtra_1.3.4                  reshape2_1.4.4                   
[17] dplyr_1.0.8       

Could you please comment on that?
Thanks,
Ali

**Comment by aheravi** [comment]
Hi Damiano,

Thanks for your prompt reply. Yes that resolved the issue.


On a different note, do you think I can reformat the mutation data I have, below, to be used in your package? I don't have the VCF files.


Thanks,

Ali



sample chr start end reference alt gene effect Amino_Acid_Change DNA_VAF SIFT PolyPhen

T1 10 123810032 123810032 A C TACC2 Missense_Mutation p.T38M 0.88 benign(0.335)



________________________________

**Comment by dami82** [comment]
Yes! This data format should be compatible with `mutSignatures`. Have a look at the vignette and make sure to set the right arguments (to match the colNames of your data.frame).
Best regards.

**Comment by cyrusmallon** [comment]
Hi,
The latest version (v2.1.5) doesn't appear to be on CRAN. Should it be?
Cheers,
Cyrus

**Comment by dami82** [comment]
Good point.It probably should. I just like GitHub better and tend to forget about CRAN.Best regards,Damiano——Damiano FantiniSent from my iPhoneOn Feb 15, 2023, at 13:19, Cyrus A. Mallon ***@***.***> wrote:﻿
Hi,
The latest version (v2.1.5) doesn't appear to be on CRAN. Should it be?
Cheers,
Cyrus

**Comment by aheravi** [comment]
Hi @dami82,
To use my mutation data and as I don't have some info based on the tutorial, I am assigning constant values to the following columns as below. Just wondering if that would be safe to do for using your tool to extract my mutation signatures.
 
QUAL = 30
FILTER = .
INFO = .
FORMAT = GT:PL
XTR1 = 0/1

Thanks.

**Comment by dami82** [comment]
Thanks for your interest and for your question.Those columns are typically found in a VCF file but they are not used by mutSignatures. So, you don't even need to create them. Best regards,Damiano——Damiano FantiniSent from my iPhoneOn Apr 13, 2023, at 18:17, aheravi ***@***.***> wrote:﻿
Hi @dami82,
To use my mutation data and as I don't have some info based on the tutorial, I am assigning constant values to the following columns as below. Just wondering if that would be safe to do for using your tool to extract my mutation signatures.
QUAL = 30
FILTER = .
INFO = .
FORMAT = GT:PL
XTR1 = 0/1
Thanks.
